README.md

```diff
@@ -1,14 +1,52 @@
 [![Build Status](https://travis-ci.com/kappnav/ui.svg?branch=master)](https://travis-ci.com/kappnav/operator)
 
-# kAppNav Helm Operator
+# kAppNav Operator
 
-Used to install Application Navigator using helm chart. The operator is to read from a custom object that represents the application navigator instance. No "helm install" is needed.
+Used to install Application Navigator. The operator is to read from a custom object that represents the application navigator instance. No "helm install" is needed.
 
 ## Build
 
+This project was developed with Operator SDK v0.10.0.
```

Comment:
Curious:  can you build and run this controller locally on your own machine, outside a container,  and not on Kubernetes?  I remember another controller I saw (a sample somewhere) could run locally.  Is that how you developed this?

Reply:
The operator's main controller loop (called reconcile) is invoked based on state changes to the CR and secondary resources it created. Need to have at least a fake Kubernetes in order to be able invoke that outside. I believe Appsody has unit tests which do just that with the Docker container. I would assume it's possible to grab the binary that the operator-sdk produced out of the temp build directory and use it directly, though I have never tried that.

Reply:
I've got Docker desktop running on my machine with Kubernetes. Build and deploy is rather quick so I wasn't tempted to cut Docker/Kubernetes out of the equation.